Tests/SonarScanner.MSBuild.PreProcessor.Test/PreProcessorTests.cs

```diff
@@ -93,9 +93,12 @@ public async Task PreProc_EndToEnd_SuccessCase()
             mockServer.AssertMethodCalled("TryGetQualityProfile", 2); // C# and VBNet
             mockServer.AssertMethodCalled("GetRules", 2); // C# and VBNet
 
-            logger.DebugMessages.Should().Contain("Base branch parameter was not provided. Incremental PR analysis is disabled.");
+            logger.AssertDebugLogged("Base branch parameter was not provided. Incremental PR analysis is disabled.");
+            logger.AssertDebugLogged("Processing analysis cache");
 
             AssertAnalysisConfig(settings.AnalysisConfigFilePath, 2, logger);
+
+            logger.AssertDebugLogged("Processing analysis cache");
```

Comment:
Should be deleted. The assert is duplicated.

Reply:
Rebase issue :/